content/tracing/product_specs/distributed_tracing.md

```diff
@@ -0,0 +1,162 @@
+    ```python
+    import asyncio
+
+    from aiohttp import web
+    from ddtrace import tracer
+    from ddtrace.contrib.aiohttp import trace_app
+
+    app = web.Application()
+    app.router.add_get('/', handle)
+
+    trace_app(app, tracer, service='thinker-microservice')
+    app['datadog_trace']['distributed_tracing_enabled'] = True
+
+    @tracer.wrap(name='think')
+    async def think(subject):
+        tracer.current_span().set_tag('subject', subject)
+
+        await asyncio.sleep(0.5)
+        return thoughts[subject]
+
+    thoughts = {
+        'technology': Thought(
+            quote='For a successful technology, reality must take precedence over public relations, for Nature cannot be fooled.',
+            author='Richard Feynman',
+        ),
+    }
+
+    async def handle(request):
+        response = {}
+        for subject in request.query.getall('subject', []):
+            await asyncio.sleep(0.2)
+            try:
+                thought = await think(subject)
+                response[subject] = {
+                    'error': False,
+                    'quote': thought.quote,
+                    'author': thought.author,
+                }
+            except KeyError:
+                response[subject] = {
+                    'error': True,
+                    'reason': 'Subject unknown'
+                }
+
+        return web.json_response(response)
+    ```
+
+The code above is already instrumented. Please [refer to the dedicated setup documentation](/tracing/setup) to learn how to instrument your application and configure the Datadog Agent.
+
+### Datadog APM
+
+Once this code is executed, [traces](tracing/services/trace/) and statistics are collected by the Agent and send to a Datadog platform in the [services list page](/tracing/services) where there are now two services: **thinker-api** and **thinker-microservice** reporting.
+
+{{< img src="tracing/miscellaneous/getting_started_distributed/services_GS.png" alt="Services list getting started" responsive="true" popup="true">}}
+
+Clicking on **thinker-api** service, directs you to its dedicated [service page](/tracing/services/service), where there are:
+
+* [Graphs illustrating service performance](/tracing/services/service/#out-of-the-box-graphs)
+* [A list of resources](/tracing/services/resource) attached to this particular service:
+
+{{< img src="tracing/miscellaneous/getting_started_distributed/resources_thinker_api_GS.png" alt="Resources thinker api getting started" responsive="true" popup="true" style="width:80%;">}}
+
+The first function executed in this example is `think_handler()`, which handles the request and forwards it to the **thinker-microservice** service.
+
+Selecting the **thinker_handler** resource directs you to its dedicated [resource page](/tracing/services/resource), where there are:
```

Comment:
"...directs you to it's automatically generated resource dashboard and a list of traces for this particular resource."

Reply:
👌